### ISSUE TITLE
Weird behaviour with horizontal dragging

### ISSUE THREAD

**Issue by gFaro** [atlassian/react-beautiful-dnd#985]
I am trying to use react-beautiful-dnd for a tab. I see a really weird behaviour when dragging a tab. The width I think is adjusted or something. You can view everything here:

https://codesandbox.io/s/23007znrzn

**Comment by alexreardon** [comment]
There is some strangeness going on with the styles. When the draggable has position:fixed applied to it and it has a with of ~400px 

You could use display: flex on the droppable to get around this
https://codesandbox.io/s/42qlz4v747

It looks like when an item from the list is removed, even when a placeholder is inserted the space collapses. I won't dig into the exact css to fix this - but that is the cause of your issue

**Comment by alexreardon** [comment]
If you need more input feel free to continue the discussion

**Comment by gFaro** [comment]
Thanks for your response. I fixed my issue with display: inline-flex and some css and html modifications. I did find it a bit weird that the js was working fine but the issues were css related, which sent me on the wrong path, but doesn't detract from an otherwise good library.

However the bigger issue is that I am building a chart component with many charts that can be dragged around horizontally or vertically inside one container. I have been able to achieve this with jquery. Is this currently possible?

I also think this can be easily achieved if instead of just reading arrays as vectors it is implemented as a matrix. This will mean you don't even have to identify whether the direction is vertical or horizontal but whatever index index in the matrix depending on the source.index and destination.index. Let me know what you think.
